packages/frontend/src/components/common/Filters/FieldAutoComplete.tsx

```diff
@@ -94,6 +102,7 @@ const FieldAutoComplete: FC<Props> = ({
                 captureDismiss: true,
             }}
             itemRenderer={renderItem}
+            activeItem={activeField}
```

Comment:
@rephus this seams to be the fix. So the component needs an activeItem and selectedItem ? 👀 🤔

Reply:
yea. and weird, prop description

    /**
     * The currently focused item for keyboard interactions, or `null` to
     * indicate that no item is active. If omitted or `undefined`, this prop will be
     * uncontrolled (managed by the component's state). Use `onActiveItemChange`
     * to listen for updates.
     */